internal/website/content/howto/runtimevar/_index.md

```diff
@@ -49,14 +89,17 @@ use it portably.
 
 The easiest way to a `Variable` is to use the [`Variable.Latest`][] method. It
 returns the latest good [`Snapshot`][] of the variable value, blocking if no
-good value has *ever* been received. The dynamic type of `Snapshot.Value`
```

Comment:
I think `good value` is more accurate than `change` here. It can have detected changes from one error to another, for example, but still not have anything to return from `Snapshot`.